src/components/select/index.js

```diff
@@ -1,5 +1,92 @@
-import ReactSelect from "react-select"
+import React from "react"
 import styled from "styled-components"
+import ReactSelect, { components } from "react-select"
+import { capitalizeFirstLetter } from "src/utils"
+
+const addDataAttrs =
+  (Component, { ga, testId }, hasInnerProps = true) =>
+  ({ innerProps, ...rest }) => {
```

Comment:
nitpick: if this is HOC, i believe the naming pattern is `with...`, ie `withDataAttrs` here